src/ResourceManager/OperationalInsights/Commands.OperationalInsights/DataSources/GetAzureOperationalInsightsDataSourceCommand.cs

```diff
@@ -57,7 +57,6 @@ public class GetAzureOperationalInsightsDataSourceCommand : OperationalInsightsB
         HelpMessage = "The data source name.")]
         [Parameter(ParameterSetName = ByWorkspaceObjectByKind)]
         [ValidateSet(
-            PSDataSourceKinds.AzureAuditLog,
```

Comment:
@haitch this is a breaking change - existing scripts that use this value for the `Kind` parameter will no longer work